src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,44 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents an find command. Upon execution, produces some feedback to the user.
+ */
+public class FindCommand extends Command {
+    private String keyword;
+
+    /**
+     * Create instance of FindCommand.
+     *
+     * @param keyword keyword to find
+     */
+    public FindCommand(String keyword) {
+        this.keyword = keyword;
+    }
+
+    /**
+     * Find the task which contains the keyword.
+     *
+     * @param tasks all of the tasks
+     * @return a TaskList of result tasks
+     */
+    private TaskList getTaskWithKeyword(TaskList tasks) {
+        TaskList result = new TaskList();
+        for (Task task : tasks.getList()) {
+            if (task.getDescription().contains(keyword)) {
+                result.addTask(task);
+            }
+        }
+        return result;
+    }
```

Comment:
this feels like it belongs in the TaskList class instead of here. just my opinion

Reply:
After reading your review, I agree with what you said. I should swift this method to my `TaskList' class. Thank you.